File: translations/codenet/deepseek_commented/C2Go/codellama/Code/s859058692.go
package main

import (
	"fmt"
	"strconv"
)

func main() {
	var sum int64
	var min, max int
	var n, m int
	fmt.Scan(&n)

	for i := 0; i < n; i++ {
		fmt.Scan(&m)
		sum += int64(m)

		if i == 0 {
			min = m
			max = m
		}

		if min > m {
			min = m
		}

		if max < m {
			max = m
		}
	}

	fmt.Println(min, max, sum)
}

File: translations/codenet/uncommented/C2P/starcoder/Code/s859058692.py

n = int(input())
sum = 0
min = 1000000000
max = -1000000000
for i in range(n):
    m = int(input())
    sum += m
    if min > m:
        min = m
    if max < m:
        max = m
print(min, max, sum)

